bm-persona/Fitness/FitnessViewController.swift

```diff
@@ -257,11 +256,11 @@ extension FitnessViewController {
     }
     
     func setupFilterTableView() {
-        let filters = [
-            Filter<Gym>(label: "Nearby", filter: Gym.locationFilter(by: Gym.nearbyDistance)),
-            Filter<Gym>(label: "Open", filter: {gym in gym.isOpen ?? false}),
+        let functions: [TableFunction] = [
+            Sort<Gym>(label: "Nearby", sort: Library.locationComparator()),
+            Filter<Gym>(label: "Open", filter: {lib in lib.isOpen ?? false}),
```

Comment:
They refer to the same thing, but I think it should be `Gym.locationComparator()`. Another inconsequential thing, but I think for this view controller, it makes more sense if `gym` is the variable name instead of `lib`, haha.

Same goes for `DiningViewController`, I guess. Just with Dining Hall themed stuff instead.

Reply:
oh yeah that was copy and paste error, forgot to change